flink-state-backends/flink-statebackend-forst/src/test/java/org/apache/flink/state/forst/ForStStateBackendConfigTest.java

```diff
@@ -779,6 +780,32 @@ public void testSupportSavepoint() {
         assertTrue(forStStateBackend.supportsSavepointFormat(SavepointFormatType.NATIVE));
     }
 
+    @Test
+    public void testConfigurePeriodicCompactionTime() throws Exception {
```

Comment:
How about testing them in `testConfigurableOptionsFromConfig` also?

Reply:
Thanks for the advice, just added.